src/utils/PayloadValidation.ts

```diff
@@ -38,6 +41,12 @@ export const validatePayload = (payload: ITechRecord, validateEntireRecord: bool
     return psvValidation.validate(payload, {abortEarly: false});
   } else if (payload.vehicleType === VEHICLE_TYPE.TRL) {
     return featureFlagValidation(trlValidation, payload, validateEntireRecord, options);
+  } else if (payload.vehicleType === VEHICLE_TYPE.LGV) {
+    return lgvValidation.validate(payload, {abortEarly: false});
+  } else if (payload.vehicleType === VEHICLE_TYPE.CAR) {
+    return carValidation.validate(payload, {abortEarly: false});
+  } else if (payload.vehicleType === VEHICLE_TYPE.MOTORCYCLE) {
+    return motorcycleValidation.validate(payload, {abortEarly: false});
```

Comment:
can we do something like
```
  const abortOptions = {abortEarly:false};
  const tankOptions = {...abortOptions , context: {isTankOrBattery}};
```
and 

`lgvValidation.validate(payload, abortOptions)`